### ISSUE TITLE
Line Wrap in Terminal Breaks Clickable URL Links

### ISSUE THREAD

**Issue by cbluth** [Microsoft/vscode#22323]
- VSCode Version: $ code -v                       
1.10.2                                                              
8076a19fdcab7e1fc1707952d652f0bb6c6db331  

- OS Version: $ uname -a                      
Linux Macbook 4.8.0-41-generic #44-Ubuntu SMP Fri Mar 3 15:27:17 U
TC 2017 x86_64 x86_64 x86_64 GNU/Linux 

Steps to Reproduce:

1. Open integrated terminal, adjust window size
2. Use terminal and output a URL, wrapping around a line

![image](https://cloud.githubusercontent.com/assets/6709636/23766060/fdfe0656-04c8-11e7-8802-693e3d6eccd4.png)

**Comment by Tyriar** [comment]
This one is blocked on https://github.com/sourcelair/xterm.js/issues/325

**Comment by jonaskello** [comment]
The upstream issue seems to be fixed [here](https://github.com/xtermjs/xterm.js/pull/616) in March 2007. 

Any progress on this issue?

**Comment by jonaskello** [comment]
Btw this issue still exists in 1.19.1.

**Comment by Tyriar** [comment]
@jonaskello that PR is only about restoring the line when it's resized down and back up again. No update on this.

Closing as duplicate of https://github.com/Microsoft/vscode/issues/28746 (easier to find in search)
